src/drivers/azure-app-configuration.ts

```diff
@@ -68,6 +68,7 @@ export default defineDriver((opts: AzureAppConfigurationOptions = {}) => {
   return {
     name: DRIVER_NAME,
     options: opts,
+    instance: getClient(),
```

Comment:
It should be a getter `getInstance`

Reply:
Then I guess I will change all of the `instance` key to `getInstance`?

Reply:
Yes please